src/django_rich/test.py

```diff
@@ -0,0 +1,132 @@
+from __future__ import annotations
+
+import sys
+import unittest
+from unittest.result import (  # type: ignore [attr-defined]
+    STDERR_LINE,
+    STDOUT_LINE,
+    failfast,
+)
+
+import django
+from django.test import testcases
+from django.test.runner import DebugSQLTextTestResult, DiscoverRunner
+from rich.color import Color
+from rich.console import Console
+from rich.rule import Rule
+from rich.style import Style
+from rich.traceback import Traceback
+
+if django.VERSION >= (3, 0):
+    from django.test.runner import PDBDebugResult
+
+
+class RichMixin(unittest.TestResult):
+    console = Console(stderr=True)
```

Comment:
I think this should possibly use `self.stream`, which is what the unittest output is being directed into - need to check more in the unittest internals

Reply:
I've currenltly changed all the outputs to write to `self.stream` as that was needed to let the `DebugSQL` tests I brought in from Django core to pass. 

Maybe I've not done this in the most DRY was as I've captured each item. (I didn't see an obvious input to Console wo which we could pass `self.stream`?)

Reply:
Yeah that wasn't what I meant. I've changed it. `self.stream` is a wrapper around a file-like object that is normally `sys.stderr`. It's that which we should pass to `Console`, so it can output to the same destination. This allows users to change the stream, say with a custom wrapper, and have rich output still.